### PR TITLE
UPSTREAM: <drop>: regenerate code

### PR DESCRIPTION
**Pull request by deads2k**
Requires https://github.com/openshift/kubernetes/pull/31 .

This is needed or status changes don't take affect

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

File: vendor/k8s.io/kubernetes/pkg/apis/extensions/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,4 +1,20 @@
-// +build !ignore_autogenerated_openshift
+// +build !ignore_autogenerated
```
Comment (sttts) [inline vendor/k8s.io/kubernetes/pkg/apis/extensions/zz_generated.deepcopy.go]: why do we get the boilerplate now? Didn't we remove that before?

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/zz_generated.deepcopy.go [hunk 2]
```diff
@@ -65,8 +65,10 @@ func DeepCopy_wardle_FlunderList(in interface{}, out interface{}, c *conversion.
 			in, out := &in.Items, &out.Items
 			*out = make([]Flunder, len(*in))
 			for i := range *in {
-				if err := DeepCopy_wardle_Flunder(&(*in)[i], &(*out)[i], c); err != nil {
+				if newVal, err := c.DeepCopy(&(*in)[i]); err != nil {
```
Comment (sttts) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/zz_generated.deepcopy.go]: Interersting that we now go through the schema. Any idea why?

**Comment by deads2k** [comment]
I just re-ran the build in openshift/kubernetes and it produced this, which I then picked here.  I didn't investigate what the generator did

File: vendor/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/zz_generated.deepcopy.go (hunk 2 above)
Comment (sttts) [inline vendor/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/zz_generated.deepcopy.go]: we have the same upstream. Maybe a regression there, not critical, just slow.

**Comment by sttts** [comment]
I was of the impression that we drop the boilerplate somewhere, but cannot find it. @stevekuznetsov do you remember?

Other than that, lgtm.

**Comment by deads2k** [comment]
> I was of the impression that we drop the boilerplate somewhere, but cannot find it. @stevekuznetsov do you remember?

Well @sttts, I found it, but it won't make you happy.  Looks like we have our own, separate, openshift generator too.  Regenerated the regenerated code.  This is nuts and its not even consistent between deep copy and conversion

File: vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go [hunk 3]
```diff
@@ -16090,16 +16090,19 @@ func (x *ISCSIVolumeSource) CodecEncodeSelf(e *codec1978.Encoder) {
 		} else {
 			yysep2 := !z.EncBinary()
 			yy2arr2 := z.EncBasicHandle().StructToArray
-			var yyq2 [7]bool
+			var yyq2 [10]bool
```
Comment (liggitt) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: wow, this was scary

Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: I have no idea what that is doing.

**Comment by stevekuznetsov** [comment]
> I was of the impression that we drop the boilerplate somewhere, but cannot find it.

> I found it, but it won't make you happy. Looks like we have our own, separate, openshift generator too. 

@deads2k @sttts I am going to need a little more background to answer this one

File: vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go (hunk 3 above)
Comment (liggitt) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: https://github.com/openshift/origin/pull/14112 picked in API changes without regenerating files.

Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: And now I do and I wish I didn't.  Ouch.

Comment (sttts) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: And we have no verify-generated-code job in CI :-/

Comment (liggitt) [inline vendor/k8s.io/kubernetes/pkg/api/v1/types.generated.go]: I thought we did for the bits we expected to generate (proto, deep copy, conversion, etc). Previously, we removed generated ugorji in origin, and so we didn't care about generating it.

**Comment by deads2k** [comment]
[merge] to unstick @DirectXMan12 

re[test]

**Comment by liggitt** [comment]
this LGTM, though we need a follow-up to remove all the types.generated.go files throughout the tree... those came in accidentally with the rebase

**Pull request merged**